specification_templates/fit.yaml

```diff
@@ -1,19 +1,10 @@
 data:
-  infection_version: '2020_06_12.10'
-  location_set_version_id: 708
+  infection_version: '2020_06_21.01'
+  location_set_version_id: 727
   output_root: ''
-  location_set_file: ''
+  location_set_file: '/ihme/covid-19/seir-pipeline-outputs/metadata-inputs/location_metadata_727.csv'
 parameters:
   n_draws: 10
-  degree: 3
```

Comment:
We gutted the spline piece of the ode fit, so these options are no longer necessary.